src/C-interface/bml_introspection.c

```diff
@@ -35,6 +35,24 @@ bml_get_type(
 
 }
 
```

Comment:
Please add documentation to this new function. If I understand it correctly then it's used to inspect the matrix types underneath the distributed2d type.